asv_bench/benchmarks/strftime.py

```diff
@@ -45,6 +48,44 @@ def time_frame_datetime_formatting_custom(self, obs):
         self.data["dt"].dt.strftime(date_format="%Y-%m-%d --- %H:%M:%S")
 
 
+class PeriodStrftime:
+    timeout = 1500
+    params = ([1000, 10000], ["D", "H"])
+    param_names = ["obs", "fq"]
+
+    def setup(self, obs, fq):
+        self.data = pd.DataFrame(
+            {
+                "p": pd.period_range(start="2000-01-01", periods=obs, freq=fq),
+                "r": [np.random.uniform()] * obs,
+            }
+        )
+
+    def time_frame_period_to_str(self, obs, fq):
+        self.data["p"].astype(str)
+
+    def time_frame_period_formatting_default(self, obs, fq):
+        """Note that as opposed to datetimes, the default format of periods are
```

Comment:
docstring->comment pls

"Periods are many and depend from the period characteristics" -> "a Period depends on its 'freq'"

Reply:
ok